.NET/Microsoft.Recognizers.Definitions.Common/Spanish/DateTimeDefinitions.cs

```diff
@@ -25,7 +25,8 @@ public static class DateTimeDefinitions
       public const bool CheckBothBeforeAfter = false;
       public const string TillRegex = @"(?<till>\b(hasta|hacia|al?)\b|~|--|-|—|——)(\s+(el|la(s)?)\b)?";
       public static readonly string RangeConnectorRegex = $@"(?<and>\b(y\s*(el|(la(s)?)?))\b|{BaseDateTime.RangeConnectorSymbolRegex})";
-      public const string DayRegex = @"(?<day>01|02|03|04|05|06|07|08|09|10|11|12|13|14|15|16|17|18|19|1|20|21|22|23|24|25|26|27|28|29|2|30|31|3|4|5|6|7|8|9)(?=\b|t)";
+      public const string WrittenDayRegex = @"(?<day>uno|dos|tres|cuatro|cinco|seis|siete|ocho|nueve|diez|once|doce|trece|catorce|quince|dieciséis|diecisiete|dieciocho|diecinueve|veinte|veintiuno|veintidós|veintitrés|veinticuatro|veinticinco|veintiséis|veintisiete|veintiocho|veintinueve|treinta|treinta y uno)";
```

Comment:
Why is this required in Spanish if there's no equivalent in English?

Reply:
We followed the Italian implementation where WrittenDayRegex is also used. We introduced this new regex there because in English these cases are covered by FlexibleDayRegex: (?<DayOfMonth>([A-Za-z]+\s)?[A-Za-z\d]+) which is constrained by (?<=(st|nd|rd|th)). But in Italian and Spanish, cardinal numbers are used instead of ordinals and without constraint the orignal regex matches too much.